driver-core/src/main/java/com/datastax/driver/core/querybuilder/Delete.java

```diff
@@ -33,21 +35,34 @@
     private boolean ifExists;
 
     Delete(String keyspace, String table, List<Selector> columns) {
-        super(keyspace);
-        this.table = table;
```

Comment:
Tried to avoid all those code duplications in the BuiltStatement class and it's children, not sure if it was worth it but it's done.